src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import IHelloProps from 'interfaces/IAppProps'
+
+export default class App extends React.Component<IHelloProps, {}>{
+    constructor(props: IHelloProps) {
```

Comment:
I don't think `constructor` is needed here.

Reply:
Yes, I will resolve it. Thanks!